lib/renderer.js

```diff
@@ -82,6 +83,18 @@ Renderer.prototype.paragraph = text => {
   return `<p>${text}</p>\n`;
 };
 
+// Prepend root to image path
+Renderer.prototype.image = function(href, title, text) {
+  const url_for = require('hexo-util').url_for.bind(this.options);
+
+  if (!parse(href).hostname && !this.options.config.relative_link
+    && this.options.prependRoot) {
+    href = url_for(href);
```

Comment:
href may be encoded when href is already encoded beforehand.
Pending https://github.com/hexojs/hexo-util/pull/92 https://github.com/hexojs/hexo-util/pull/94

Reply:
Resolved in https://github.com/hexojs/hexo-util/pull/99